From a4fd9d55f87ea9ece2d7ea6bb183db7877b12d11 Mon Sep 17 00:00:00 2001
From: Neal <neal_liu@aspeedtech.com>
Date: Fri, 12 Sep 2025 02:11:58 +0000
Subject: [PATCH 26/32] configs: evb-ast2700: enable FIT signature configure

Enable FIT signature configure.

Change-Id: I36830236fa7e5ae54bf8721a209ee61ab47d684b
Signed-off-by: Neal Liu <neal_liu@aspeedtech.com>
(cherry picked from commit 883d3304542e9bdf003209463d8c846be8429f63)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 configs/evb-ast2700_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/evb-ast2700_defconfig b/configs/evb-ast2700_defconfig
index 64be06df3e..c8fca7c57d 100644
--- a/configs/evb-ast2700_defconfig
+++ b/configs/evb-ast2700_defconfig
@@ -31,7 +31,9 @@ CONFIG_SYS_MEMTEST_START=0x403000000
 CONFIG_SYS_MEMTEST_END=0x403001000
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_FIT=y
+CONFIG_FIT_SIGNATURE=y
 # CONFIG_BOOTMETH_EFILOADER is not set
+CONFIG_LEGACY_IMAGE_FORMAT=y
 CONFIG_USE_BOOTARGS=y
 CONFIG_BOOTARGS="console=ttyS12,115200n8 root=/dev/ram rw earlycon"
 CONFIG_USE_BOOTCOMMAND=y
-- 
2.34.1

